### DISCUSSION TITLE
Eglot version "from future" in ELPA (eglot v1.10)

### DISCUSSION THREAD

**Discussion by eugeneandrienko** [joaotavora/eglot#1158] [General]
Hello. Today I've started to configure eglot to work with Java via Eclipse JDT language server. And suddenly found what in ELPA there is a Eglot v1.10. There are no signs of release 1.10 in this repository, so this is very suspicious. The latest official version is 1.9. Source code of eglot v1.10 differs from eglot v1.9.

This "future" version of eglot was not compatible with eglot-java and caused strange error. When I do a [little debug](https://github.com/yveszoundi/eglot-java/issues/8) — I found problem with "unofficial" version from future.

Maybe eglot v1.10 was published by mistake? Can it be removed from ELPA to not confusing new users — I afraid what not all emacs packages based on eglot works with 1.10 version (like eglot-java).

**Comment by joaotavora** [comment]
Are you aware of this: https://github.com/joaotavora/eglot#get-latest-development-version ?

**Reply by eugeneandrienko** [reply]
Oh missed that — I've searched only by version number (1.10). Thank you!
